### PR TITLE
enable Scribunto to wiki

### PR DESCRIPTION
**Pull request by ImBoPhil**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -272,10 +272,11 @@
 		'mecanonwiki' => true, 
 		'quantixwiki' => true,
 	),
-	'wmgUseScribunto' => array(
```
Comment (Reception123) [inline LocalSettings.php]: please put the ( back

File: LocalSettings.php [hunk 2]
```diff
@@ -272,10 +272,11 @@
...
+	'wmgUseScribunto' => array
 		'default' => false,
 		'allthetropeswiki' => true,
 		'allthetropestestwiki' => true,
+		'aryamanwiki => true,'
```
Comment (Southparkfan) [inline LocalSettings.php]: 'aryamanwiki => true,'
should be:
'aryamanwiki' => true,

File: LocalSettings.php [hunk 3]
```diff
@@ -276,6 +276,7 @@
 		'default' => false,
 		'allthetropeswiki' => true,
 		'allthetropestestwiki' => true,
+		'aryamanwiki => true,
```
Comment (Reception123) [inline LocalSettings.php]: 'aryamanwiki => true,
please add this 
'aryamanwiki' => true,

**Comment by NDKilla** [comment]
@ImBoPhil Welcome to feature requests. :) going to go ahead and merge this now.

**Pull request merged**